shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -79,13 +79,18 @@ void PlatformViewChannel::ClearViewFactories() {
   view_factories_.clear();
 }
 
-void PlatformViewChannel::SendKeyEvent(Ecore_Event_Key* event, bool is_down) {
+void PlatformViewChannel::SendKeyEvent(const char* key,
+                                       const char* string,
+                                       const char* compose,
+                                       uint32_t modifiers,
+                                       uint32_t keycode,
+                                       bool is_down) {
   PlatformView* view = FindFocusedView();
   if (view) {
     if (is_down) {
-      view->DispatchKeyDownEvent(event);
+      view->DispatchKeyDownEvent(key, string, compose, modifiers, keycode);
```

Comment:
I would like to consider this topic along with [this](https://github.com/flutter-tizen/engine/pull/291#discussion_r884389195).

Reply:
Ok.
Changing the name of the `DispatchKeyDownEvent` was not a consideration for me.

This interface was created to support `webview` implementation in Tizen, and because these names are derived from the names of the widely known webview interface and their internal methods.

public boolean dispatchKeyEvent ([KeyEvent](https://developer.android.com/reference/android/view/KeyEvent) event)

Furthermore, I've never seen a place where 'Hook' is used except in the Windows world.

Reply:
IMO, ``DispatchKeyDownEvent`` is not so bad. 
At the same time, I think @swift-kim 's suggestion can be accepted.
( ``KeyboardHook`` is a bit less familiar... to me)

Reply:
I suggest using `SendKey' as a name universally except for the callback function.